modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/document/library/DocumentLibraryDDMFormFieldTemplateContextContributor.java

```diff
@@ -305,6 +307,31 @@ private FileEntry _getFileEntry(JSONObject valueJSONObject) {
 		}
 	}
 
+	private Map<String, Object> _getFileEntryDeleteParameters(
+		DDMFormFieldRenderingContext ddmFormFieldRenderingContext) {
+
+		HttpServletRequest httpServletRequest =
+			ddmFormFieldRenderingContext.getHttpServletRequest();
+
+		return HashMapBuilder.<String, Object>put(
+			"fileEntryDeleteURL",
+			() -> {
+				RequestBackedPortletURLFactory requestBackedPortletURLFactory =
```

Comment:
@icgsbr please also move the `httpServletRequest` to be inside, just like the `requestBackedPortletURLFactory`